quizbizz/src/App.jsx

```diff
@@ -8,6 +8,7 @@ import MockMan from "mockman-js";
 import QuizInstruction from "./pages/quizinstruction/QuizInstruction";
 import RequiredAuth from "./components/require-auth/RequiredAuth";
 import QuizQuestion from "./pages/quizquestion/QuizQuestion";
+import QuizAnswers from "./pages/quizanswers/QuizAnswers";
```

Comment:
Suggestion: you can import all files in index.js file and use it from there

Reply:
ok